bisect_ppx.opam

```diff
@@ -22,7 +22,7 @@ depends: [
   "cmdliner" {>= "1.0.0"}
   "dune" {>= "2.7.0"}
   "ocaml" {>= "4.03.0"}
-  "ppxlib" {>= "0.21.0"}
+  "ppxlib" {>= "0.26.0" & < "0.29.0"}
```

Comment:
FYI it appears that after this PR, the right upper constraint is `< "0.28.0"`:

```
File "src/ppx/instrument.mli", line 8, characters 11-57:
8 |    inherit Ppxlib.Ast_traverse.map_with_expansion_context
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error: Unbound class type Ppxlib.Ast_traverse.map_with_expansion_context
```

However, I'll probably try to port Bisect to 0.28.0 rather than add the constraint. @pitag-ha, do you know if people need a release of Bisect that is compatible with ppxlib 0.26.0 and 0.27.0, at this point?

Reply:
> FYI it appears that after this PR, the right upper constraint is < "0.28.0":

Good catch, thanks!

>  @pitag-ha, do you know if people need a release of Bisect that is compatible with ppxlib 0.26.0 and 0.27.0, at this point?

I think `ppxlib >= 0.28.0` support directly should be enough.